client/src/components/views/Planning.jsx

```diff
@@ -0,0 +1,86 @@
+import { Heading, SimpleGrid, Text, Center, Box } from '@chakra-ui/react';
+import useWindowSize from 'hooks/useWindowSize';
+import { productionData, plData } from 'components/Planning/PlanningData.ts';
+import Head from 'next/head';
+import React from 'react';
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  Legend,
+  Tooltip,
+  XAxis,
+  YAxis,
+  // eslint-disable-next-line import/no-unresolved
```

Comment:
I think this line can be removed, if you have errors from ESLint in your IDE about this, try running `npm i` inside the client folder (not in the container).